### PR TITLE
fix[#950] add missing `~`

### PR DESCRIPTION
**Pull request by wgx731**
Also included a `myBids` as in memory storage for easy testing

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @wgx731,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by jlprat** [comment]
OK TO TEST

File: docs/src/test/scala/docs/http/scaladsl/HttpServerExampleSpec.scala [hunk 1]
```diff
@@ -529,11 +529,21 @@ class HttpServerExampleSpec extends WordSpec with Matchers
       case object GetBids
       case class Bids(bids: List[Bid])
 
-      class Auction extends Actor {
-        def receive = {
-          case Bid(userId, bid) => println(s"Bid complete: $userId, $bid")
-          case _ => println("Invalid message")
-        }
+      class Auction extends Actor {
```
Comment (jlprat) [inline docs/src/test/scala/docs/http/scaladsl/HttpServerExampleSpec.scala]: As a note, this change is also addressed by another PR (#933)

Comment (wgx731) [inline docs/src/test/scala/docs/http/scaladsl/HttpServerExampleSpec.scala]: done

Comment (jlprat) [inline docs/src/test/scala/docs/http/scaladsl/HttpServerExampleSpec.scala]: :+1:

**Comment by jlprat** [comment]
Refs: #950

**Pull request merged**